plugin.json

```diff
@@ -109,19 +109,19 @@
           "default": false,
           "help_text": "(Optional) When set to true it enables the call recordings functionality."
         },
+        {
+          "key": "JobServiceURL",
+          "display_name": "Job service URL",
+          "type": "text",
+          "help_text": "The URL to a running calls job service instance used for call recordings.",
```

Comment:
@streamer45, would something like "The URL of the server you're using to store call recordings" be accurate in this case?

Reply:
Not really as recordings are ultimately uploaded to the Mattermost server and that's where they will live. This service is where recording files are produced since it would be too expensive (computation wise) to do it on the Mattermost instance so we require to deploy this dedicated service to execute recording jobs (that deal with a/v/ transcoding and such).